dev/source/docs/sitl-serial-mapping.rst

```diff
@@ -1,28 +0,0 @@
-.. _sitl-serial-mapping:
```

Comment:
normally we do not delete pages, rather, mark them archived, and include in the archive page list...

also, you missed a reference to this page in  using-sitl-for-ardupilot-testing, which needs to be updated